server/src/views/main.tmpl

```diff
@@ -4,7 +4,7 @@
 
   <!-- Header -->
   <header id="header">
-    <h1>Hanabi Live</h1>
+    <h1><a href="/">Hanabi Live</a></h1>
```

Comment:
it should't be on main because that will fuck up the user

Reply:
why? it's just a reload

Reply:
if the user is on a slow computer or a slow internet connection, one stray misclick will cause a frustrating 30 second period where they cannot do anything anymore. that is terrible